Graphics/GraphicsEngineOpenGL/interface/RenderDeviceGLES.h

```diff
@@ -56,8 +61,14 @@ DILIGENT_BEGIN_INTERFACE(IRenderDeviceGLES, IRenderDeviceGL)
 
     VIRTUAL void METHOD(Suspend)(THIS) PURE;
 
+#if PLATFORM_ANDROID
     VIRTUAL EGLint METHOD(Resume)(THIS_
                                   ANativeWindow* window) PURE;
+#elif PLATFORM_EMSCRIPTEN
+// Not supported for EMSCRIPTEN
+#else
+#    error Unsupported platform
+#endif
```

Comment:
This is Android specific, may be it should be
#if PLATFORM_ANDROID
    VIRTUAL EGLint METHOD(Resume)(THIS_
                                  ANativeWindow* window) PURE;
#endif